docs/user-manual/modules/ROOT/pages/index.adoc

```diff
@@ -14,6 +14,8 @@ routing rules in your IDE, whether in a Java or XML editor.
 
 For a deeper and better understanding of Apache Camel, an xref:faq/what-is-camel.adoc[Introduction] is provided.
 
+Furthermore, camel has added a new API i.e. Endpoint-dsl. More information can be founded under documentation section.
```

Comment:
"Can be found" should be better I guess

Reply:
Yes, this should be "can be found".I'll fix it

Reply:
I have made the change in the file